asv_bench/asv.conf.json

```diff
@@ -41,7 +41,10 @@
         "sqlalchemy": [],
         "scipy": [],
         "numexpr": [],
-        "pytables": [],
+        "tables": [],
```

Comment:
Why did you change ``pytables`` to ``tables``? It doesn't work for me know on ``conda`` environment. It ask if I meant ``pytables``.